src/network/peers.rs

```diff
@@ -671,7 +665,13 @@ impl<N: Network, E: Environment> Peers<N, E> {
     ///
     /// Sends the given message to every connected peer, excluding the sender.
     ///
-    async fn propagate(&self, sender: SocketAddr, message: Message<N, E>) {
```

Comment:
what do you say about this?

`warning: variable does not need to be mutable
   --> src/network/peers.rs:672:51
    |
672 |     async fn propagate(&self, sender: SocketAddr, mut message: Message<N, E>) {
    |                                                   ----^^^^^^^
    |                                                   |
    |                                                   help: remove this `mut`
    |
    = note: `#[warn(unused_mut)]` on by default

warning: `snarkos` (lib) generated 1 warning`